### DISCUSSION TITLE
Git Source Control does not show Modified files

### DISCUSSION THREAD

**Discussion by rgilotra** [coder/code-server#6109] [Q&A]
I have installed git and git base extension on code-server instance from vsx-gallery. But after installation and cloning the git directory. Source Control does not show modified files. 
code-server Version: 3.10.2
Any Suggestion why i am not able to see modified files in source control ?
![source-control](https://user-images.githubusercontent.com/30127391/228000738-a435c7bf-5b40-4983-bad9-954069d3be2c.PNG)

**Comment by code-asher** [comment]
Does this happen with 4.11.0 as well?  Are there any logs (from either the browser console or in the logs for git in the output panel).

**Reply by rgilotra** [reply]
Thanks much. Its working after updating it to 4.11.0 version.
